test/presubmit-tests.sh

```diff
@@ -85,10 +85,23 @@ api-unittest(){
   info Running unittests
 
   go mod vendor
-  go test -p 1 -v ./pkg/... && 
+  go test -p 1 -v ./pkg/... &&
   go test -p 1 -v ./v1/service/...
 }
 
+api-golangci-lint() {
+  info Running go lint
+
+  go mod vendor
```

Comment:
is go mod vendor required again?  `api-unittest` is ran before this and that has go mod vendor.